apps/questapi/moon.yml

```diff
@@ -0,0 +1,19 @@
+tasks:
+  install:
+    command:
+      npm install --save-dev
+  dev:
+    command: 
+      npm run dev
+  build:
+    command:
+      npm run build
+  start:
+    command:
+      npm start
+  generateprismaclient:
```

Comment:
can we do `db:gen:client` or just `db:client` ? `generateprismaclient` is too long. 

We have to use double quote like this: `"db:gen:client"` since yaml doesn't accept `:` for the key.